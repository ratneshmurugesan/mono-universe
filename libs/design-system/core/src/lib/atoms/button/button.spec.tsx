import { render } from '@testing-library/react';

import Button from './button';

describe('Button', () => {
  it('should render successfully', () => {
    const { baseElement } = render(<Button>Base button for test</Button>);
    expect(baseElement).toBeTruthy();
  });
});
