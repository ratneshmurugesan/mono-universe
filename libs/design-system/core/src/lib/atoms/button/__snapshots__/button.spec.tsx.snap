// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render with a snapshot 1`] = `
<button
  class="sc-bczRLJ"
  type="button"
>
  Base button
</button>
`;
